cachito/web/app.py

```diff
@@ -29,7 +30,12 @@ def healthcheck():
     current_app.logger.info("A healthcheck request was received")
 
     try:
+        start_time = timer()
+
         db.session.execute("SELECT 1 FROM request LIMIT 0").fetchall()
+
+        end_time = (timer() - start_time) * 1000
+        current_app.logger.info("The healthcheck database query took %0.3f milliseconds", end_time)
```

Comment:
Based on https://docs.python.org/3/library/timeit.html#timeit.default_timer, `default_timer` is this thing https://docs.python.org/3/library/time.html#time.perf_counter, which counts in seconds. Unless I am missing something, multiplying seconds by 1000 will not give you milliseconds  :D

Reply:
Well, 1000 milliseconds == 1 second, right? I got me so unsure of it that I went to look it up online :sweat_smile: 

Anyway, the only reason I did this is because the results are quite often fractions of a second, so I thought it would look better on the logs. We could simply log in seconds, if you prefer ;)

Reply:
Why convert to milliseconds? How about just log the time difference directly? According to the livenessProbe config, 0.xxxxx should always be the expected value.

Reply:
Oh right, I was indeed missing something :D
Yeah, we can just keep it in seconds

Reply:
Done :D